Illustrations/Accuracy_vs_Sex feature/Sex_feature_vs_Accuracy.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier
+binwidth=0.2
```

Comment:
The code is neat and really good.
I appreciate it @vishal1383 .

If you can add some line spaces in between the main code and the imports , it will do the job :1st_place_medal:

Reply:
Sure I will modify them.